include/guestbook.theme.inc

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @file
+ * The implementation of the theme functions.
+ */
+
+/**
+ * Generates messages html markup.
+ *
+ * @return
+ *   Html markup.
+ */
+function guestbook_html() {
+  global $user;
+
+  // Variable $mes is fully generated html markup.
+  $mes = NULL;
```

Comment:
what is  variables `mes`? The name of variable should reflect its purpose, use message instead